trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core/src/org/eclipse/tracecompass/incubator/internal/trace/server/jersey/rest/core/model/TreeDataModel.java

```diff
@@ -34,8 +34,8 @@ public interface TreeDataModel {
     /**
      * @return The parent ID.
      */
-    @Schema(description = "Unique id to identify this parent's entry, " +
-            "optional if this entry does not have a parent.")
+    @Schema(description = "Optional unique id to identify this parent's entry. " +
+            "Use -1 if this entry has no parent. If omitted, this entry has no parent.")
```

Comment:
It feels like we are encouraging the use of -1 when omitting would be just as good. Maybe I would suggest:
"If the parentId is -1 or omitted, this entry has no parent".

Reply:
Done